docs/src/introduction.md

```diff
@@ -58,6 +59,32 @@ p4est_connectivity_destroy(connectivity)
 ```
 
 
+## [`PointerWrapper`s](@id pointer_wrappers)
+
+As we have seen in the [previous section](@ref translation_guidelines) many functions
```

Comment:
```suggestion
As we have seen in the previous section on [translation guidelines](@ref translation_guidelines), many functions
```
Suggestion: This way we do not assume that people read everything from top to bottom, therefore giving them some more context